local-maps-server/services/osm.ts

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { hostUrl } from "../common";
+
+export const osm = {
+  tilejson: "2.0.0",
+  name: "OSM",
+  attribution:
+    '<a href="https://www.openstreetmap.org/copyright" target="_blank">&copy; OpenStreetMap contributors</a>',
+  minzoom: 0,
+  maxzoom: 22,
```

Comment:
what happens if i have only zoom up to 8?, will that be a problem?

Reply:
So when set maximum zoom to 22 here, if import tiles set is zoom 0-8, when users click into zoom 9, the tile map will be empty, and they can click zoom back to 8. The reason I used a maximum 22 is that users won't be limited whatever zoom level tiles set they imported.

Also we can provide a default value here and users can define the maximum zoom value by a docker env var when they run the server container.

Reply:
Got it.